apps/explorer/src/components/Layout/PageLayout.tsx

```diff
@@ -13,6 +13,24 @@ import Header from '../header/Header';
 import { useNetworkContext } from '~/context';
 import { Banner } from '~/ui/Banner';
 import { Network } from '~/utils/api/DefaultRpcClient';
+import suiscanImg from '~/assets/explorer-suiscan.jpg';
+import suivisionImg from '~/assets/explorer-suivision.jpg';
+import suiscanImg2x from '~/assets/explorer-suiscan@2x.jpg';
+import suivisionImg2x from '~/assets/explorer-suivision@2x.jpg';
+import { ButtonOrLink } from '~/ui/utils/ButtonOrLink';
+import { Image } from '~/ui/image/Image';
+import { ArrowRight12, Sui, SuiLogoTxt } from '@mysten/icons';
+import { useRedirectExplorerUrl } from '~/hooks/useRedirectExplorerUrl';
+
+enum RedirectExplorer {
+	SUISCAN = 'suiscan',
+	SUIVISION = 'suivision',
+}
+
+const isSuiVisionFirst = new Date().getMilliseconds() % 2 === 0;
```

Comment:
Can we persist this for each user so that it doesn't flip all the time?

Reply:
Updated to use localStorage